packages/core/src/command/consolidated/get-snapshot-local.ts

```diff
@@ -125,3 +133,25 @@ async function postSnapshotToApi({
     }
   );
 }
+
+export function removeContainedPatients(bundle: Bundle): Bundle {
+  if (!bundle.entry) return bundle;
+
+  const updatedEntry = bundle.entry.map(entry => {
+    if (entry.resource && "contained" in entry.resource) {
+      return {
+        ...entry,
+        resource: {
+          ...entry.resource,
+          contained: entry.resource.contained?.filter(r => !isPatient(r)),
```

Comment:
this is removing all patients. I thought we said to remove patients whose IDs arent matching the target patient ID?